static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
```

Comment:
Are we suggesting `lazydom` as a name for a promise wrapped version of `fastDom`? I think it's better to be explicit. I've seen promisfy used as terminology before, perhaps `promisifiedFastDom`, but it is a bit long. My point is explicitness is better as it won't cause confusion.